src/components/EmbedFile/index.js

```diff
@@ -7,13 +7,16 @@ import CodeBlock from '@theme/CodeBlock';
 const guessLanguageFromFilePath = (filePath) => {
     if (/\.ya?ml$/i.test(filePath)) {
         return 'yaml';
-    }
+    }   
```

Comment:
```suggestion
    }
```

Reply:
I think we chose to remove embed file since it was locking up the app build for local testing